components/automate-cli/pkg/verifyserver/server/api/v1/systemresources.go

```diff
@@ -0,0 +1,24 @@
+package v1
+
+import (
+	"net/http"
+
+	"github.com/chef/automate/components/automate-cli/pkg/verifyserver/response"
+	"github.com/gofiber/fiber"
+)
+
+func (h *Handler) GetSystemResource(c *fiber.Ctx) {
+
+	nodeType := c.Query("node_type", "")
+	deploymentState := c.Query("deployment_state", "")
```

Comment:
You can also check if the query is empty or not valid with code base

Reply:
Done